.github/workflows/dartpad_ui.yml

```diff
@@ -35,6 +35,9 @@ jobs:
       - run: flutter pub get
       - run: flutter analyze
       - run: dart format --set-exit-if-changed .
+      - name: Create template projects and populate the custom pub cache
+        run: dart run grinder build-project-templates
+        working-directory: pkgs/dart_services/
```

Comment:
In this GitHub action, we're creating the project templates, but we're not running the server? Why is that?

Reply:
Nevermind, I see we are doing this in presubmit/main_test, which will run on line 42.